packages/gestalt/src/RadioButton.css

```diff
@@ -1,4 +1,8 @@
 :root {
+  --border-sm: 6px;
+  --border-md: 8px;
```

Comment:
@AlbertCarreras The disabled state doesn't look correct - we should also update the label color:

Your branch:
![image](https://user-images.githubusercontent.com/127199/77102195-abd47700-69d5-11ea-86c5-541c5d89d65b.png)

Design:
![image](https://user-images.githubusercontent.com/127199/77102149-a0814b80-69d5-11ea-93b0-b6c58f5b7be7.png)

Reply:
Damn, I didn't notice that!

Reply:
Done

Reply:
Done